sdk/core/azure-core/src/main/java/com/azure/core/implementation/SwaggerMethodParser.java

```diff
@@ -53,6 +54,7 @@
  * method.
  */
 public class SwaggerMethodParser implements HttpResponseDecodeData {
+    private final ClientLogger logger = new ClientLogger(SwaggerMethodParser.class);
```

Comment:
There doesn't appear to be any use of this logger. Is it needed.

Reply:
Removed.